src/QMCWaveFunctions/TWFFastDerivWrapper.h

```diff
@@ -27,6 +34,8 @@ namespace qmcplusplus
  *
  *  Please see : J. Chem. Phys. 144, 194105 (2016) https://doi.org/10.1063/1.4948778 for implementation details and formalism.
  */
+struct TWFFastDerivWrapperMultiWalkerMem;
```

Comment:
Please move this class into the TWFFastDerivWrapper class right above `mw_mem_handle_` definition.

Reply:
no idea what you mean..

Reply:
It is currently a free standalone class. It can be better if it becomes a member of TWFFastDerivWrapper.

Reply:
This is what I meant moving the definition. 01afc2c868a51ccac83ecbbfc740f104f6f22cd0